examples/puzzletron/distillation/README.md

```
@@ -0,0 +1,186 @@
+    --student-mcore-checkpoint /path/to/kd-checkpoints/iter_0000400 \
+    --output-hf-checkpoint /path/to/exported-hf
+```
+
+The exported directory will contain the converted weights plus the tokenizer / config
+files copied verbatim from `--student-hf-checkpoint`
+(`config.json`, `tokenizer.json`, `tokenizer_config.json`, `special_tokens_map.json`,
+`chat_template.jinja`).
+
+## Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)